addons/web_tour/static/src/js/tip.js

```diff
@@ -316,8 +316,25 @@ var Tip = Widget.extend({
                 of: this.$anchor,
                 collision: "none",
                 using: props => {
-                    this.el.style.setProperty('top', `${props.top}px`, 'important');
-                    this.el.style.setProperty('left', `${props.left}px`, 'important');
+                    const {top} = props;
+                    let {left} = props;
+                    const anchor = this.$anchor[0];
+                    if (['P', 'H1', 'H2', 'H3', 'H4', 'H5', 'H6'].includes(anchor.nodeName)) {
```

Comment:
Add that list as a property of the class (next to events, etc). That way it can be extended and it will be more clear what that is (name it something like `CENTER_ON_TEXTS_TAGS`).